tests/wpt/meta/html/semantics/forms/the-input-element/files.html.ini

```diff
@@ -1,7 +1,5 @@
 [files.html]
   type: testharness
-  [setting <input type=file>.files]
-    expected: FAIL
```

Comment:
There are two tests failing so I was expecting to see two test expectations files updated. In addition, you'll need to do the same set of updates for the legacy layout system (just pass `--legacy-layout` to all `mach` commands which should be visible above).

Reply:
Ah apologies - I was getting timeouts when running those wpt commands earlier, so tried updating manually - but it seems to be working for me now.

Reply:
Oh yeah, I have noticed these timeouts as well when running with the `--legacy-layout` flag. It would be nice to get to the bottom of it.